scripts/convert_lit_checkpoint.py

```diff
@@ -196,6 +196,19 @@ def maybe_unwrap_state_dict(lit_weights: Dict[str, torch.Tensor]) -> Dict[str, t
     return lit_weights.get("model", lit_weights)
 
 
+def maybe_raise_finetune_warning(lit_weights: Dict[str, torch.Tensor]) -> None:
```

Comment:
This says warning but it's an exception, I suggest a different name:

```suggestion
def check_conversion_supported(lit_weights: Dict[str, torch.Tensor]) -> None:
```